src/index.tsx

```diff
@@ -17,6 +19,13 @@ const { store, persistor } = redux;
 
 const queryClient = new QueryClient();
 
+Sentry.init({
+  environment: process.env.NODE_ENV,
```

Comment:
Perhaps, choosing to set NODE_ENV to staging may not be the optimal choice. Ideally, we would prefer to have NODE_ENV=production in the staging environment, as we aim for staging to closely resemble production. How do you feel about creating a distinct ENV specifically for defining the Sentry environment? https://glebbahmutov.com/blog/do-not-use-node-env-for-staging/